p/themes/Origine/origine.css

```diff
@@ -2,10 +2,19 @@
 
 /*=== FONTS */
 @font-face {
-	font-family: "OpenSans";
-	src: url("../fonts/openSans.woff") format("woff");
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/opensans.eot'); /* IE9 Compat Modes */
+  src: local('Open Sans'), local('OpenSans'),
+       url('../fonts/opensans.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
```

Comment:
We do not support IE < 10 anyway, due to other technical limitations, so I guess this line can be removed.